pkg/grizzly/providers.go

```diff
@@ -131,10 +136,12 @@ func (r *Resource) MatchesTarget(targets []string) bool {
 	if len(targets) == 0 {
 		return true
 	}
-	key := r.Key()
+	UID := r.UID()
+	slashKey := fmt.Sprintf("%s/%s", r.Kind(), UID)
+	dotKey := fmt.Sprintf("%s.%s", r.Kind(), UID)
```

Comment:
I'd be tempted to use the following here, to have less duplication and fewer sources of truth:

```suggestion
	dotKey := r.Key() // fmt.Sprintf("%s.%s", r.Kind(), UID)
	slashKey := fmt.Sprintf("%s/%s", r.Kind(), UID)
```

Reply:
done